File: 2390-removing-stars-from-a-string/2390-removing-stars-from-a-string.c
char * removeStars(char * s){
    char * stack =(char *) malloc((strlen(s)+1)*sizeof(char));
    int c =0,len= strlen(s);
    
    for(int i =0; i<len; i++)
    {
        if(s[i] == '*'){
            if(c==0){ continue; }
           else{ c -= 2; }
        } else {
            stack[c]=s[i];
        }
        c++;
    }
    stack[c]='\0';
    return stack;
}

File: 2390-removing-stars-from-a-string/2390-removing-stars-from-a-string.js
/**
 * @param {string} s
 * @return {string}
 */
var removeStars = function(s) {
    const stack = [];

    for (const c of s) {
        if (c === '*' && stack.length) {
            stack.pop();
        }
        else {
            stack.push(c);
        }
    }

    return stack.join('');
};